lib/app.js

```diff
@@ -222,6 +223,17 @@ var makeApp = function(configBase, callback) {
 
         app.config = config;
 
+        app.on("close",function(){
+            clearInterval(dialbackClient.cleanup);
+            if(nonceCleanerInterval){
```

Comment:
Small style nit: can you do `if () {` instead of `if(){`, for consistency?

Same for the `if` clause on line 231.